### PR TITLE
N°7078 Hungarian translations

### PR DESCRIPTION
**Pull request by piRGoif**
Done by @tacsaby

Translations on iTop were pushed with Combodo/iTop#584

### CODE REVIEW

File: hu.dict.combodo-webhook-integration.php [hunk 1]
```diff
@@ -2,9 +2,10 @@
 /**
  * Localized data
  *
- * @copyright   Copyright (C) 2022
+ * @copyright   Copyright (C) 2017 ITOMIG GmbH
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:5]: This isn't right

File: hu.dict.combodo-webhook-integration.php [hunk 2]
```diff
@@ -61,9 +62,6 @@
 	'Class:ActionWebhook/Attribute:method/Value:put' => 'PUT',
 	'Class:ActionWebhook/Attribute:method/Value:patch' => 'PATCH',
 	'Class:ActionWebhook/Attribute:method/Value:delete' => 'DELETE',
-	'Class:ActionWebhook/Attribute:method/Value:head' => 'HEAD~~',
-	'Class:ActionWebhook/Attribute:path' => 'Path~~',
-	'Class:ActionWebhook/Attribute:path+' => 'Additional path to append to the connection URL (eg. \'/some/specific-endpoint\')~~',
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:66]: Please revert, the one for `Value:head` can be `HEAD`.

File: hu.dict.combodo-webhook-integration.php [hunk 3]
```diff
@@ -75,15 +73,15 @@
 
 2 féle módszert használhat:
 - A kiváltó objektumból (pl. UserRequest), nyilvánosnak kell lennie. Példa: $this->XXX($aContextArgs, $oLog, $oAction)
-- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
+- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:76]:
```suggestion
- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```

File: hu.dict.combodo-webhook-integration.php [hunk 4]
```diff
@@ -75,15 +73,15 @@
...
 
 FONTOS: Ha be van állítva, az \'Adatfolyam\' attribútumot figyelmen kívül hagyjuk.',
 	'Class:ActionWebhook/Attribute:process_response_callback' => 'Folyamat válasz visszahívás',
 	'Class:ActionWebhook/Attribute:process_response_callback+' => 'PHP-módszer a webhook-hívás válaszának feldolgozására.
 
 2 típusú metódust használhat:
 - Magából a kiváltó objektumból (pl. UserRequest), nyilvánosnak kell lennie. Példa: $this->XXX($oResponse, $oAction)
-- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $oResponse, $oAction)
+- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \SomeClass::XXX($oObject, $oResponse, $oAction)
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:84]:
```suggestion
- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $oResponse, $oAction)
```

File: hu.dict.combodo-webhook-integration.php [hunk 5]
```diff
@@ -119,7 +117,7 @@
 
 2 típusú metódust használhat:
 - Magából a kiváltó objektumból (pl. UserRequest), nyilvánosnak kell lennie. Példa: $this->XXX($aContextArgs, $oLog, $oAction)
-- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
+- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:120]:
```suggestion
- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```

File: hu.dict.combodo-webhook-integration.php [hunk 6]
```diff
@@ -157,7 +155,7 @@
...
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:158]:
```suggestion
- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```

File: hu.dict.combodo-webhook-integration.php [hunk 7]
```diff
@@ -185,12 +183,12 @@
...
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:186]:
```suggestion
- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```

File: hu.dict.combodo-webhook-integration.php [hunk 8]
```diff
@@ -185,12 +183,12 @@
...
 
 FONTOS: Ha be van állítva, a \'Üzenet\' és az összes \'Bot információ\' figyelmen kívül marad.',
 	// - Fieldsets
 	'ActionRocketChatNotification:message' => 'Bázisüzenet',
-	'ActionRocketChatNotification:additionalelements' => 'Bot információ',
+	'ActionRocketChatNotification:additionalelements' => 'Bot információ'
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:191]:
```suggestion
	'ActionRocketChatNotification:additionalelements' => 'Bot információ',
```

File: hu.dict.combodo-webhook-integration.php [hunk 9]
```diff
@@ -204,7 +202,7 @@
 
 2 típusú metódust használhat:
 - Magából a kiváltó objektumból (pl. UserRequest), nyilvánosnak kell lennie. Példa: $this->XXX($aContextArgs, $oLog, $oAction)
-- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
+- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:205]:
```suggestion
- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```

File: hu.dict.combodo-webhook-integration.php [hunk 10]
```diff
@@ -245,7 +243,7 @@
...
```
Comment (Hipska) [inline hu.dict.combodo-webhook-integration.php:246]:
```suggestion
- Bármely PHP osztályból, statikusnak ÉS nyilvánosnak kell lennie. A névnek teljesen minősített névnek kell lennie. Példa: \\SomeClass::XXX($oObject, $aContextArgs, $oLog, $oAction)
```

**Review by Hipska — CHANGES_REQUESTED** [review]

**Comment by Molkobain** [comment]
Accepted during functional review

**Review by Molkobain — CHANGES_REQUESTED** [review]
Nothing more than HIpska, he pointed it all out 😁

File: hu.dict.combodo-webhook-integration.php (hunk 1 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 2 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 3 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 4 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 5 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 6 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 7 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 8 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 9 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

File: hu.dict.combodo-webhook-integration.php (hunk 10 above)
Comment (piRGoif) [reply hu.dict.combodo-webhook-integration.php]: Fixed in d824e455

**Comment by piRGoif** [comment]
About \SomeClass VS \\SomeClass : what I see in the UI is the following

image

I will replace all values to remove the backslash :)

**Comment by piRGoif** [comment]
Wooops after a chat with Guillaume, revert modification so that `\SomeClass::...` would appear anywhere
The goal is to make visible it is a FQCN that should be set

**Pull request merged**